dotcom-rendering/.storybook/decorators/splitThemeDecorator.tsx

```diff
@@ -14,20 +18,25 @@ const headerCss = css`
 `;
 
 const splitCss = css`
-	width: 50%;
 	padding: ${space[4]}px;
 `;
 
 const darkStoryCss = css`
-	float: right;
 	background-color: ${sourcePalette.neutral[0]};
 	color: ${sourcePalette.neutral[100]};
 `;
 const lightStoryCss = css`
-	float: left;
 	background-color: ${sourcePalette.neutral[100]};
 	color: ${sourcePalette.neutral[0]};
 `;
+const rowCss = css`
+	display: flex;
+	flex-direction: row;
+`;
+const columnCss = css`
+	display: flex;
+	flex-direction: column;
+`;
```

Comment:
[As per the Emotion Best Practice, dynamic values should be set using the `style` prop](https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles).

Not a blocker as we do not follow this everywhere.